components/shopify_partner/sources/new-app-charges/new-app-charges.mjs

```diff
@@ -5,7 +5,7 @@ export default {
   key: "shopify_partner-new-app-charges",
   name: "New App Charges",
   type: "source",
-  version: "0.0.14",
+  version: "0.0.15",
```

Comment:
```suggestion
  version: "0.1.0",
```

Reply:
This source does not use the appId prop which is what was updated on this PR, hence why it only receives a minor version bump